packages/app/src/debug/DebugFailedTest.cy.tsx

```diff
@@ -199,4 +182,37 @@ describe('<DebugFailedTest/>', () => {
 
     cy.percySnapshot()
   })
+
+  it('conditionally renders artifacts', () => {
+    const render = (testResult: TestResults) => cy.mount(() =>
+      <DebugFailedTest failedTestsResult={[testResult]} groups={[group1]} expandable={false}/>)
```

Comment:
```suggestion
    const render = (testResult: TestResults) => cy.mount(() =>
      (<div data-cy="test-group">
        <DebugFailedTest failedTestsResult={[testResult]} groups={[group1]} expandable={false}/>
      </div>))
```
Only one minor suggestion which is to wrap the component with the same data-cy attribute to allow the styling to show the buttons on hover.